website/docs/Building-Premake.md

```diff
@@ -7,7 +7,7 @@ If you downloaded a prebuilt binary package you can skip this page, which discus
 
 ## Using a Source Code Package ##
 
-If you have one of the [official source code packages](http://premake.github.io/download.html), you'll find that project files for a variety of toolsets have already been generated for you in the `build/` folder. Find the right set for your platform and toolset and build as you normally would  (i.e. run `make`). The resulting binaries will be placed in the top-level `bin/` folder.
+If you have one of the [official source code packages](/download), you'll find that project files for a variety of toolsets have already been generated for you in the `build/` folder. Find the right set for your platform and toolset and build as you normally would  (i.e. run `make`). The resulting binaries will be placed in the top-level `bin/` folder.
```

Comment:
Just to confirm, this change is correct? (I can see that it works in your demo)

Reply:
Yes. This link `http://premake.github.io/download.html` is old and incorrect. Download page is stored in `website/src/pages/download.js` so it's completely valid to reference it using `/download`. I fixed it on the fly when I was reviewing all usages of markdown links in our docs.

Reply:
Thanks for this explanation!